### PR TITLE
feat(pwa): change default icons

### PR DESCRIPTION
**Pull request by userquin**
This PR includes:
- change default pwa icons to use transparent backgrond color
- add 64x64 icon for windows
- reduce padding in maskable-icon
- added how to generate icons in docs
- update pwa plugin to latest (0.15.1)
- todo to add related apps (`modules/pwa/i18n.ts`)

### CODE REVIEW

**Comment by cyberalien** [comment]
Looks good on iOS:

Simulator Screenshot - iPhone 14 Pro Max - 2023-05-25 at 20 17 01

File: modules/build-env.ts [hunk 1]
```diff
@@ -29,8 +29,8 @@ export default defineNuxtModule({
 
     nuxt.options.nitro.publicAssets = nuxt.options.nitro.publicAssets || []
     if (env === 'dev')
-      nuxt.options.nitro.publicAssets.unshift({ dir: resolve('../public-dev') })
+      nuxt.options.nitro.publicAssets.unshift({ dir: resolve('../public-dev'), baseURL: '/' })
```
Comment (userquin) [inline modules/build-env.ts:32]: upps, remove baseURL from here and below

**Comment by userquin** [comment]
I'll change the alpha mask in Elk logos, right now using `fill: '#D9D9D9'`, all generated icons have some transparency, changing the color to white seems to fix it (I need to regenerate all icons):
```
<mask
     id="a"
     width="240"
     height="234"
     x="4"
     y="1"
     maskUnits="userSpaceOnUse"
     style="mask-type:alpha">
    <path
       fill="#D9D9D9"
       d="M244 123c0 64.617-38.383 112-103 112-64.617 0-103-30.883-103-95.5C38 111.194-8.729 36.236 8 16 29.46-9.959 88.689 6 125 6c64.617 0 119 52.383 119 117Z"
       id="path19" />
  </mask>
```

**Comment by userquin** [comment]
don't merge yet, I need to optimize png files...

**Comment by userquin** [comment]
we should regenerate `favicon.ico` (transparent 64x64 32 bit color), I haven't the tools on my laptop to do it.

**Comment by userquin** [comment]
Once #2130 ready, this PR can be closed

**Comment by userquin** [comment]
supersed by #2130

**Pull request closed without merging**